source/blood/src/demo.cpp

```diff
@@ -406,7 +406,7 @@ void CDemo::Playback(void)
                 if (v4 >= atf.nInputCount)
                 {
                     ready2send = 0;
-                    if (at59ef != 1)
+                    if (at59ef > 1)
```

Comment:
rename at59ef to demoCount ?

Reply:
That's done in another RP actually. https://github.com/nukeykt/NBlood/pull/549

Reply:
👍

Reply:
Would you mind renaming the variable to match #549 if you don't mind please? I'm not particularly familiar with Git so I'd probably end up messing something up if I tried to change this PR myself (if I even can)! 

Thanks for all your help so far :)

Reply:
No problem, hopefully this is all that needed to be changed to resolve the merge conflict. Thanks!

Reply:
Hmm no still get a conflict. I could just manually push the change as a separate commit?

Reply:
I'm not sure myself, I've merged with master on my end so yeah you may have to do that.